plugins/vercel/api_token_test.go

```diff
@@ -0,0 +1,56 @@
+package vercel
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAPITokenProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(
+		t, APIToken().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+			"default": {
+				ItemFields: map[sdk.FieldName]string{
+					fieldname.Token: "tZk79pLyPLGgUVlkHbnLeXgl",
+				},
+				ExpectedOutput: sdk.ProvisionOutput{
+					CommandLine: []string{"--token", "tZk79pLyPLGgUVlkHbnLeXgl"},
+				},
+			},
+		},
+	)
+}
+
+func TestAPITokenImporter(t *testing.T) {
+	plugintest.TestImporter(
+		t, APIToken().Importer, map[string]plugintest.ImportCase{
+			"environment": {
+				Environment: map[string]string{
+					"VERCEL_TOKEN": "tZk79pLyPLGgUVlkHbnLeXgl",
+				},
+				ExpectedCandidates: []sdk.ImportCandidate{
+					{
+						Fields: map[sdk.FieldName]string{
+							fieldname.Token: "tZk79pLyPLGgUVlkHbnLeXgl",
+						},
+					},
+				},
+			},
+			"config file": {
+				OS: "darwin",
```

Comment:
Let's also add a case for the Linux importer, to cover all the possible options here. 😄